relayer/application-standard.ts

```diff
@@ -40,7 +41,8 @@ const defaultOpts: Partial<StandardRelayerAppOpts> = {
   workflows: {
     retries: 3,
   },
-  fetchSourceTxhash: true
+  fetchSourceTxhash: true,
+  logger: defaultLogger,
```

Comment:
If we don't have this we get an error if the opt is not specified

Reply:
Yeah actually I think there are 2 things here:
- by default we should still log in a sane way
- the user should be able to turn logging off entirely and not cause an exception 

So imo we should still have the above code, but additionally track down the source of the exceptions. Thoughts?

Reply:
this is already merged, but for whatever is worth:
I think that this module shouldn't log anything by default. In the case we receive a logger, then use that logger to log, which is a way of saying "pass a logger to turn on the logs".
If we receive no logger, then we instantiate a logger mock that doesn't log, and then we don't have to worry about the exceptions. Is this pattern to ugly?